Source/CombatExtended/CombatExtended/Verbs/Verb_LaunchProjectileCE.cs

```diff
@@ -348,9 +359,128 @@ public virtual void ShiftTarget(ShiftVecReport report, bool calculateMechanicalO
                     }
                     else if (currentTarget.Thing is Pawn)
                     {
-                        // Aim for center of mass on an exposed target
-                        targetRange.min = victimVert.BottomHeight;
+                        targetRange.min = victimVert.BottomHeight;
                         targetRange.max = victimVert.MiddleHeight;
+
+                        bool AppropiateAimMode = CompFireModes?.CurrentAimMode != AimMode.SuppressFire;
+
+                        bool flagTargetMode = (CompFireModes?.targetMode != TargettingMode.automatic);
+
+                        bool isPlayerFaction = (CasterPawn?.Faction ?? Caster.Faction) == Faction.OfPlayer;
+
+                        bool flag1 = ((ShootingAccuracy >= 2.2f) | isTurret);
```

Comment:
When I see `bool flag1 = ...`, I am immediately looking for the second place it is used, or why it must be calculated here but used later.  Since it is used immediately, and only once, there is no reason to store it in a flag variable.  If you don't like the line length from `if ((ShootingAccuracy >= 2.2f) | isTurret)` (which is shorter than the `bool flag1 = ` line), then go ahead and split it onto two lines.

```C#
if ((ShootingAccuracy >= 2.2f) |
     isTurret);
```